### ISSUE TITLE
Pre-work Checklist: Developer: MJ Shelton

### ISSUE THREAD

**Issue by mjshelton12** [hackforla/website#4163]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by mjshelton12** [comment]
1. Progress: I am about 1/3 of the way done
2. Blockers: No blockers currently
3. Availability: I have my weekend wide open and ready to work on this - about 12hrs each day though I don't anticipate needing that
4. ETA: By Monday EOD at the latest
5. Pictures (optional): N/A

**Comment by mjshelton12** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

About 4hrs including all onboarding activities, spread out over a few days (I am now 5 days out from initial onboarding). Maybe 2hrs on just this issue.

**Comment by mjshelton12** [comment]
Checklist is complete. Keeping this open until I can complete the following complexity of issues:

- Good Second Issue (ETA hopefully this week)
- Small Issue (Pending completing previous issue)
- Medium Issue (Pending completing previous issue)
- Large Issue (Pending completing previous issue)

**Comment by mjshelton12** [comment]
Updated

Issues worked on:

- Good First Issue
- Good Second Issue
- Small Issue

Issues to work on:

- Good Medium Issue
- Good Large Issue

Total PRs reviewed to date: 4

**Comment by mjshelton12** [comment]
Update 3/31

Issues worked on:

Good First Issue
Good Second Issue
Small Issue

Currently working on: Medium Issue (expected to complete 4/16 due to need to present in all-team meeting)

Issues to work on:

Large Issue

Total PRs reviewed to date: 8

**Comment by mjshelton12** [comment]
**Update 4/16**

**Issues worked on:**
Good First Issue
Good Second Issue
Small Issue

**Currently working on:** 
Medium Issue (expected to complete by 4/23 provided dev leads approve of my work)

**Issues to work on:**
Large Issue

**Total PRs reviewed to date:**
10+ (I've lost count but I know I've done more than 10 which is the threshold to get to [here](https://github.com/hackforla/website/wiki/Being-a-Part-of-the-Hack-For-LA--Dev-Team))

**Comment by mjshelton12** [comment]
Update 4/24

Issues worked on:
Good First Issue
Good Second Issue
Small Issue
Medium Issue (my medium issue was changed to a large issue after more work was added so I'm going to count that as having done a medium)

Currently working on:
Large Issue

Issues to work on:
None

Total PRs reviewed to date:
10+ (I've lost count but I know I've done more than 10 which is the threshold to get to [here](https://github.com/hackforla/website/wiki/Being-a-Part-of-the-Hack-For-LA--Dev-Team))

**Comment by mjshelton12** [comment]
Update 5/8

Issues worked on:
Good First Issue
Good Second Issue
Small Issue
Medium Issue (my medium issue was changed to a large issue after more work was added so I'm going to count that as having done a medium)

Currently working on:
Large Issue

Issues to work on:
None

Total PRs reviewed to date:
10+ (I've lost count but I know I've done more than 10 which is the threshold to get to [here](https://github.com/hackforla/website/wiki/Being-a-Part-of-the-Hack-For-LA--Dev-Team))

Other updates:
Since last updating, I joined the Merge Team :)

**Comment by roslynwythe** [comment]
Hi @mjshelton12.  We ask that if you are not working on other issues, please leave an update at least every two weeks in this issue, to avoid having the `2 weeks inactive' label applied, which flags the item for review by a dev lead.  If you will be away from Hack for LA for a period of time, we can move the issue out of "In Progress" for a specific period of time, so that the `2 weeks inactive` label will not be applied.  Please let us know what your intentions are.  Thanks!

**Comment by HackforLABot** [comment]
## Activity Log: mjshelton12
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- mjshelton12 opened: https://github.com/hackforla/website/issues/4163 at 03/13/2023, 08:04 PM PDT
- mjshelton12 assigned: https://github.com/hackforla/website/issues/4163 at 03/13/2023, 08:10 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/issues/4163#issuecomment-1474659029 at 03/17/2023, 08:20 PM PDT
- mjshelton12 assigned: https://github.com/hackforla/website/issues/3856#issuecomment-1425070509 at 03/18/2023, 04:38 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/issues/3856#issuecomment-1475042456 at 03/18/2023, 05:14 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/issues/4163#issuecomment-1475064805 at 03/18/2023, 06:45 PM PDT
- mjshelton12 opened: https://github.com/hackforla/website/pull/4210 at 03/18/2023, 07:00 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4203#issuecomment-1475378919 at 03/19/2023, 12:48 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4203#pullrequestreview-1347594009 at 03/19/2023, 03:33 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4213#issuecomment-1475455415 at 03/19/2023, 05:28 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4213#pullrequestreview-1347678236 at 03/19/2023, 06:36 PM PDT
- mjshelton12 PR was merged: https://github.com/hackforla/website/pull/4210#event-8790513238 at 03/19/2023, 10:59 PM PDT
- mjshelton12 issue closed by PR 4210: https://github.com/hackforla/website/issues/3856#event-8790513417 at 03/19/2023, 10:59 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/issues/4163#issuecomment-1478823844 at 03/21/2023, 07:08 PM PDT
- mjshelton12 assigned: https://github.com/hackforla/website/issues/4219 at 03/21/2023, 08:15 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/issues/4219#issuecomment-1478865242 at 03/21/2023, 08:18 PM PDT
- mjshelton12 opened: https://github.com/hackforla/website/pull/4234 at 03/21/2023, 09:55 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/issues/4219#issuecomment-1478918514 at 03/21/2023, 09:58 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4260#issuecomment-1481866658 at 03/23/2023, 01:40 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4260#pullrequestreview-1355819113 at 03/23/2023, 04:55 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4265#issuecomment-1482086992 at 03/23/2023, 05:30 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4265#pullrequestreview-1355847745 at 03/23/2023, 05:33 PM PDT
- mjshelton12 assigned: https://github.com/hackforla/website/issues/4238 at 03/23/2023, 05:35 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/issues/4238#issuecomment-1482092458 at 03/23/2023, 05:36 PM PDT
- mjshelton12 opened: https://github.com/hackforla/website/pull/4267 at 03/23/2023, 06:03 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/issues/4163#issuecomment-1482109733 at 03/23/2023, 06:05 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4272#issuecomment-1482153074 at 03/23/2023, 07:09 PM PDT
- mjshelton12 PR was merged: https://github.com/hackforla/website/pull/4234#event-8842483275 at 03/24/2023, 01:24 PM PDT
- mjshelton12 issue closed by PR 4234: https://github.com/hackforla/website/issues/4219#event-8842483499 at 03/24/2023, 01:24 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4272#pullrequestreview-1357614619 at 03/24/2023, 05:33 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4275#issuecomment-1483635567 at 03/24/2023, 05:34 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4267#issuecomment-1483715096 at 03/24/2023, 09:17 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4275#pullrequestreview-1357773011 at 03/24/2023, 09:34 PM PDT
- mjshelton12 PR was merged: https://github.com/hackforla/website/pull/4267#event-8847606258 at 03/26/2023, 09:47 AM PDT
- mjshelton12 issue closed by PR 4267: https://github.com/hackforla/website/issues/4238#event-8847606286 at 03/26/2023, 09:47 AM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4292#issuecomment-1484502299 at 03/26/2023, 10:07 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4292#pullrequestreview-1358339382 at 03/26/2023, 10:14 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4290#issuecomment-1484512421 at 03/26/2023, 10:18 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4260#pullrequestreview-1359987552 at 03/27/2023, 04:09 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4290#pullrequestreview-1361960981 at 03/28/2023, 02:55 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4308#issuecomment-1487866400 at 03/28/2023, 07:39 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4308#pullrequestreview-1363677262 at 03/29/2023, 10:57 AM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/issues/4163#issuecomment-1492681645 at 03/31/2023, 03:32 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4353#issuecomment-1492758913 at 03/31/2023, 05:34 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4353#pullrequestreview-1367804328 at 03/31/2023, 08:47 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4412#issuecomment-1498489603 at 04/05/2023, 09:53 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4412#pullrequestreview-1374052267 at 04/05/2023, 10:00 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4425#issuecomment-1498496171 at 04/05/2023, 10:03 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4425#pullrequestreview-1375721275 at 04/06/2023, 06:00 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4435#issuecomment-1499816144 at 04/06/2023, 06:17 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4520#issuecomment-1510488683 at 04/16/2023, 02:09 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4520#pullrequestreview-1387055077 at 04/16/2023, 02:21 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/issues/4163#issuecomment-1510497075 at 04/16/2023, 02:47 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4548#issuecomment-1515428491 at 04/19/2023, 02:48 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4548#pullrequestreview-1392950044 at 04/19/2023, 03:09 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4546#issuecomment-1516752937 at 04/20/2023, 11:15 AM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4546#pullrequestreview-1394582955 at 04/20/2023, 11:51 AM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4554#issuecomment-1518500084 at 04/21/2023, 09:03 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4554#pullrequestreview-1396556909 at 04/21/2023, 09:15 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4496#issuecomment-1519113740 at 04/23/2023, 10:15 AM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/issues/4163#issuecomment-1520946670 at 04/24/2023, 04:25 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4496#issuecomment-1529337695 at 04/30/2023, 09:31 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/issues/4163#issuecomment-1539272353 at 05/08/2023, 06:44 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4608#issuecomment-1539275243 at 05/08/2023, 06:50 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4608#issuecomment-1539319983 at 05/08/2023, 08:02 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4608#pullrequestreview-1423674704 at 05/11/2023, 07:13 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4600#issuecomment-1550476762 at 05/16/2023, 04:17 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4600#pullrequestreview-1429586912 at 05/16/2023, 04:45 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4668#issuecomment-1550499028 at 05/16/2023, 04:55 PM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4668#pullrequestreview-1431852217 at 05/17/2023, 05:35 PM PDT
- mjshelton12 commented on: https://github.com/hackforla/website/pull/4701#issuecomment-1556219556 at 05/21/2023, 09:28 AM PDT
- mjshelton12 submitted review: https://github.com/hackforla/website/pull/4701#pullrequestreview-1442638791 at 05/24/2023, 12:48 PM PDT
